client/src/i18n/en.json

```diff
@@ -240,7 +253,13 @@
       "REQUIRED"           : "This value is required."
     },
     "INFOS": {
+      "BY"                   : "By",
+      "CLOSE_FISCAL_YEAR"    : "Close fiscal year",
+      "CLOSED"               : "Closed",
+      "CREATE"               : "CREATE",        
```

Comment:
Does this already exist somewhere?  It seems to be that "Create" is a translation key we use a lot.